modules/dbnd/src/dbnd/_core/task_run/task_run_sync_local.py

```diff
@@ -50,11 +52,8 @@ def __init__(self, task_run):
 
     def _local_cache_target(self, target_):
         # type: (FileTarget) -> FileTarget
-        return target(
-            self.task_run.attemp_folder_local_cache,
-            os.path.basename(target_.path),
-            config=target_.config,
-        )
+        path = urlparse.urlparse(target_.path).path.lstrip("/")
```

Comment:
```
>>> a = "s3://my_bucket/my_value"
>>> urlparse.urlparse(a).path.lstrip("/")
'my_value'
>>>
```
Is this the expected result? I thought we discussed keeping the remote hierarchy, in this example the value I was expecting is "my_bucket/my_value". Did I misunderstand? :)

Reply:
You understood well, however I'm not sure if this is expected result or not. Result of `urlparse(path).path` is not the same ad `os.path.basename` because if we had path `s3://my_bucket/my_dir/my_val` it would return `my_dir/my_val` instead of `my_val` and this is what we want. I'm not sure though if we want to "cut off" `my_bucket` - I think that with what I had in mind when I started writing this code we should preserve bucket name as we want our clients to be able to see exactly what they cached locally and they might have similar dirs in different buckets.

Reply:
ok I think I managed to fix it once and for all @jonathanshir